Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -332,3 +332,28 @@
 "home-nearby-footer" = "More from nearby your location";
 "home-nearby-nothing" = "Nothing nearby";
 "home-nearby-check-again" = "Check again";
+
+"welcome-whats-new-title" = "Welcome To Wikipedia";
+"welcome-whats-new-whats-new" = "WHAT'S NEW";
```

Comment:
@montehurd Improved the code comment to explain why. Lets see if that works for them. This way we get upper case capitalizations in translations if they are appropriate for a language. If we still get blow back we can go back to doing it in code.